framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -294,14 +297,17 @@ public void parse(Map<Element, AnnotatedTypeMirror> atypes, Map<String, Set<Anno
         parse(this.index, atypes, declAnnos);
     }
 
-    private void parse(IndexUnit index, Map<Element, AnnotatedTypeMirror> atypes, Map<String, Set<AnnotationMirror>> declAnnos) {
```

Comment:
I personally keep the signature on one line for easier textual searching, but we can make the change if you prefer.  (It does avoid the super-long lines.)